[32mINFO    [0m integration.ha_tests.test_async_replication:test_async_replication.py:86 Destroying second model
[32mINFO    [0m pytest_operator.plugin:plugin.py:903 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.4.5    unsupported  01:52:45Z

App                  Version  Status  Scale  Charm                Channel      Rev  Address         Exposed  Message
postgresql-k8s       14.13    active      3  postgresql-k8s                      0  10.152.183.30   no       Primary
postgresql-test-app           active      1  postgresql-test-app  latest/edge  305  10.152.183.225  no       received database credentials of the first database

Unit                    Workload  Agent      Address       Ports  Message
postgresql-k8s/0*       active    executing  10.1.246.206         Primary
postgresql-k8s/1        active    executing  10.1.246.205         
postgresql-k8s/2        active    executing  10.1.246.204         
postgresql-test-app/0*  active    idle       10.1.246.213         received database credentials of the first database

Offer              Application     Charm           Rev  Connected  Endpoint           Interface         Role
replication        postgresql-k8s  postgresql-k8s  0    0/0        replication        postgresql_async  requirer
replication-offer  postgresql-k8s  postgresql-k8s  0    1/1        replication-offer  postgresql_async  provider

[32mINFO    [0m pytest_operator.plugin:plugin.py:909 Juju error logs:

unit-postgresql-k8s-2: 01:35:58 ERROR unit.postgresql-k8s/2.juju-log pgdata folder not found in /var/lib/postgresql/data/pgdata

[32mINFO    [0m pytest_operator.plugin:plugin.py:991 Forgetting model main...